login functionality + sql injection protection function

ivailo-devrix opened the pull request.

ivailo-devrix committed:
login functionality + sql injection protection function

Neychok commented on includes/theme-compat/header.php:
```diff
@@ -1,4 +1,12 @@
 <?php
+// server should keep session data for AT LEAST 1 hour
+ini_set('session.gc_maxlifetime', 3600);
+
+// each client should remember their session id for EXACTLY 1 hour
+session_set_cookie_params(3600);
```
I think those should be in the `config.php` file

ivailo-devrix replied:
This is currently a test functionality. I've moved it to the mandatory includes file "required-includes.php" that loads on every page. 

In the config file, my idea is to have only and only constants. My idea is to keep it as clean as possible and to have absolutely no functionality in it.

Neychok reviewed (changes requested):
Just one small thing and we can merge this.

ivailo-devrix committed:
validate-domain

add a function that returns a valid domain

ivailo-devrix committed:
Create form.php

place where information submitted from forms will be processed

ivailo-devrix committed:
Update .gitignore

ivailo-devrix committed:
Update register.php

adding dynamic data to Register Form

ivailo-devrix committed:
add function extract_valid_phone

add function extract_valid_phone
edit documentation
update extract_valid_domaine function

ivailo-devrix committed:
Update form.php

implement new function extract_valid_phone

ivailo-devrix committed:
Update register.php

displaying the value entered by the client and not the prepared validated  value for the DB

ivailo-devrix committed:
remove comment html / add standardized data to DB / check admin privileges

ivailo-devrix committed:
fixed some bugs

Neychok reviewed (approved):
Let's get this puppy going.

metodiew commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
 .idea/
 .work/
 img/company/
+test.php
```
@yanachkov-devix what would happen if you happen to have a `test.php` file in your Unit tests in the future?

metodiew commented on includes/db-connect.php:
```diff
@@ -21,15 +22,38 @@ function open_db_conn()
 }
 
 
+//Preventing SQL Injection Attacks in Postgres
```
I did not know we are using Postgres heres🙈
